packages/main/src/plugin/updater.ts

```diff
@@ -26,6 +27,7 @@ import {
   type UpdateDownloadedEvent,
   type UpdateInfo,
 } from 'electron-updater';
+import * as appAdapter from 'electron-updater/out/AppAdapter.js';
```

Comment:
Why importing the all package?

```suggestion
import { getAppCacheDir } from 'electron-updater/out/AppAdapter.js';
```

importing using `*` prevent tree shaking